### DISCUSSION TITLE
ignorePath override for subpath

### DISCUSSION THREAD

**Discussion by JerkyTreats** [renovatebot/renovate#32849] [Request Help]
### How are you running Renovate?

A Mend.io-hosted app

### Please tell us more about your question or problem

Given the following folder structure, is there a way to add an exception to `ignorePaths: tools` for `tools/do_not_ignore`?
```
tools/to_ignore_1
tools/to_ignore_2
tools/do_not_ignore
```

```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:recommended"
  ],
  "ignorePaths": [
      "tools"
  ],
```

I've tried adding `includesPath` with the subpath, but having both include/ignore appears to break config and renovate finds no packages. 
I've also tried `"!tools/do_not_ignore"` but that also appears to break config with the same behavior of no found packages. 

This appears to work:
```
  "ignorePaths": [
      "tools/to_ignore_1"
      "tools/to_ignore_2"
  ],
```

But it sucks if there are many tools to ignore. 

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
You only _need_ to use `ignorePaths` if you want them skipped from the extract phase altogether. i.e. you want to pretend that they're not even there. 

If instead all you want is to _not update them_ then you can use `packageRules` with `matchFiles` instead. Write one rule which has matchFileNames=tools/**/* + enabled=false, and a second rule which says matchFileNames=tools/do_not_ignore/**/* and enabled=true. The last rule will win, and re-enable the do_not_ignore one.
